Makefile

```diff
@@ -36,7 +36,7 @@ build:
 	           --workdir $(CURDIR) \
 	           -e GOOS=linux \
 	           -e GOARCH=amd64 \
-	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) go build -v -o $(BIN_DIR)/$(CHECKUP_IMAGE_NAME) ./cmd/
+	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) go build -buildvcs=false -v -o $(BIN_DIR)/$(CHECKUP_IMAGE_NAME) ./cmd/
```

Comment:
This change doesn't seem to be related to the PR.

Reply:
nope